saleor/thumbnail/utils.py

```diff
@@ -50,7 +50,7 @@ def get_thumbnail_size(size: Union[str, int]) -> int:
 def prepare_thumbnail_file_name(
     file_name: str, size: int, format: Optional[str]
 ) -> str:
-    file_path, file_ext = file_name.rsplit(".")
+    file_path, file_ext = file_name.rsplit(".", 1)
```

Comment:
Is it possible to pass a file without an extension?

Reply:
This function uses already uploaded files, so I think those should always have extensions because we validate them in mutations.

Reply:
We would need to check so we are actually sure of it

Reply:
confirmed @NyanKiyoshi